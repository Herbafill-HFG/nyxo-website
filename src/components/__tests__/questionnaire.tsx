import React from "react"
import renderer from "react-test-renderer"

import QuestionnaireCard from "../questionaireCard"

describe("Questionnaire", () => {
  // Mock data for testing
  const title = "Some title"
  const id = "1234556789"

  it("renders correctly", () => {
    const tree = renderer
      .create(<QuestionnaireCard name={title} path={id} />)
      .toJSON()
    expect(tree).toMatchSnapshot()
  })
})
