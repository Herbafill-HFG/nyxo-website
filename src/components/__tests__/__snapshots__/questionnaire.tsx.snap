// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Questionnaire renders correctly 1`] = `
<a
  className="sc-AxjAm cvbjvD"
  href="1234556789"
>
  <h4
    className="sc-AxirZ fXFnpQ"
  >
    Some title
  </h4>
</a>
`;
